types/scaleway-functions/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "npm-naming": false
```

Comment:
I don't think you need to disable this rule. See comment about the header above.